components/General/Faram/validations.js

```diff
@@ -95,16 +96,10 @@ export const lengthEqualToCondition = n => (value) => {
     };
 };
 
-
 export const requiredCondition = (value) => {
-    const ok = isTruthy(value) && !(
-        (
-            typeof value === 'string' &&
-            splitInWhitespace(value).length <= 0
-        ) || (
-            Array.isArray(value) &&
-            value.length <= 0
-        )
+    const ok = isDefined(value) && !(
```

Comment:
Maybe better to use `isTruthy(value, [true, false, ''])` that `isDefined(value)`